src/grid.tsx

```diff
@@ -59,6 +59,181 @@ if ((module as any).hot) {
+ *      @param {Array} [options.header.complexColumns] - This options creates new parent headers of the multiple columns
+ *          which includes the headers of spcified columns, and sets up the hierarchy.
+ *      @param {boolean} [options.virtualScrolling=false] - If set to true, use virtual-scrolling so that large
+ *          amount of data can be processed performantly. When using this option that sets true, the rowHeight option
+ *          must set value.
+ *      @param {string|number} [options.rowHeight] - The height of each rows. The default value is 'auto',
+ *          the height of each rows expands to dom's height. If set to number, the height is fixed.
+ *      @param {number} [options.minRowHeight=40] - The minimum height of each rows. When this value is larger than
+ *          the row's height, it set to the row's height.
+ *      @param {string|number} [options.bodyHeight] - The height of body area. The default value is 'auto',
+ *          the height of body area expands to total height of rows. If set to 'fitToParent', the height of the grid
+ *          will expand to fit the height of parent element. If set to number, the height is fixed.
+ *      @param {number} [options.minBodyHeight=minRowHeight] - The minimum height of body area. When this value
+ *          is larger than the body's height, it set to the body's height.
+ *      @param {Object} [options.columnOptions] - Option object for all columns
+ *      @param {number} [options.columnOptions.minWidth=50] - Minimum width of each columns
+ *      @param {boolean} [options.columnOptions.resizable=true] - If set to true, resize-handles of each columns
+ *          will be shown.
+ *      @param {number} [options.columnOptions.frozenCount=0] - The number of frozen columns.
+ *          The columns indexed from 0 to this value will always be shown on the left side.
+ *          {@link Grid#setFrozenColumnCount} can be used for setting this value dynamically.
+ *      @param {number} [options.columnOptions.frozenBorderWidth=1] - The value of frozen border width.
+ *          When the frozen columns are created by "frozenCount" option, the frozen border width set.
+ *      @param {Object} [options.treeColumnOptions] - Option object for the tree column.
+ *      @param {string} [options.treeColumnOptions.name] - The name of column that makes tree column.
+ *      @param {boolean} [options.treeColumnOptions.useIcon=true] - If set to true, the folder or file icon is created on
+ *          the left side of the tree cell data.
+ *      @param {boolean} [options.treeColumnOptions.useCascadingCheckbox] - If set to true, a cascading relationship is
+ *          created in the checkbox between parent and child rows.
+ *      @param {Object} [options.copyOptions] - Option object for clipboard copying
+ *      @param {boolean} [options.copyOptions.useFormattedValue] - Whether to use formatted values or original values
+ *          as a string to be copied to the clipboard
```

Comment:
```
 *      @param {boolean} [options.copyOptions.useListItemText] - Copy select or checkbox cell values to `text` 
 *          rather than` value` of listItem
 *      @param {string|function} [options.copyOptions.customValue] - Copy text with `formatter` in cell
```